altair/vegalite/v3/display.py

```diff
@@ -73,6 +73,12 @@ def svg_renderer(spec, **metadata):
                               vegaembed_version=VEGAEMBED_VERSION,
                               vegalite_version=VEGALITE_VERSION)
 
+zeppelin_renderer = HTMLRenderer(mode='vega-lite',
+                              fullhtml=True, requirejs=False,
+                              vega_version=VEGA_VERSION,
+                              vegaembed_version=VEGAEMBED_VERSION,
+                              vegalite_version=VEGALITE_VERSION)
```

Comment:
Fix indentation to line up with ``mode`=`

Reply:
Addressed

Reply:
Thanks